config/version-compatibility-matrix.yaml

```diff
@@ -47,6 +47,15 @@
 # The Compatible Version Matrix between Maistra and Kiali (there is a 1-to-1 relationship between Kiali and Maistra versions)
 - meshName: "Maistra"
   versionRange: 
+    - meshVersion: "2.2"
+      kialiMinimumVersion: "1.48"
+      kialiMaximumVersion: ""
+    - meshVersion: "2.2"
+      kialiMinimumVersion: "1.36"
+      kialiMaximumVersion: ""
+    - meshVersion: "2.2"
+      kialiMinimumVersion: "1.24"
+      kialiMaximumVersion: ""
     - meshVersion: "2.1"
```

Comment:
I don't totally get this structure for maistra and ossm, how is this different from one entry with 1.24 as the Minimum Version?

Reply:
Because there is a strict 1-to-1 mapping between kiali and OSSM. You don't want to say "2.2 has a minimum of 1.24" because that implies 1.25 is supported, and it is not. The next supported version is 1.36. Hence the empty "Maximum" version which is handled specially in the code. I don't recall this code explicitly so can't describe it in detail, but that's how I remember it working.